extensions/wikia/FounderEmails/events/FounderEmailsRegisterEvent.class.php

```diff
@@ -28,60 +30,58 @@ public function enabled ( $wikiId, User $user ) {
 
 	public function process( Array $events ) {
 		global $wgEnableAnswers, $wgCityId;
-		wfProfileIn( __METHOD__ );
 
```

Comment:
All the below changes mostly represent a indention change.  I updated that ```isThresholdMet``` check with a simple count( $events ) check and return early if necessary.